developer_manual/app/fundamentals/info.rst

```diff
@@ -29,7 +33,8 @@ In this section, you will find a complete example configuration, along with an e
           <developer>https://doc.owncloud.org</developer>
       </documentation>
 
-      <category>tool</category>
+      <author>Your Name</author>
```

Comment:
This is not required by the marketplace but could be mandatory for owncloud (not sure)